codespell_lib/data/dictionary.txt

```diff
@@ -24834,9 +24911,14 @@ fonctionnalities->functionalities
 fonctionnality->functionality
 fonctionnaly->functionally, functionality,
 fonctions->functions
+fondamental->fundamental
+fondamentally->fundamentally
+fondamentals->fundamentals
 fonetic->phonetic
 fontain->fountain, contain,
 fontains->fountains, contains,
+fontend->frontend, contend,
+fontends->frontends, contends,
```

Comment:
Perhaps add `fontended` and `fontending`, unless they are too rare.

Reply:
These are too rare.